pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Namespaces.java

```diff
@@ -1361,4 +1361,39 @@ SchemaAutoUpdateCompatibilityStrategy getSchemaAutoUpdateCompatibilityStrategy(S
     void setSchemaAutoUpdateCompatibilityStrategy(String namespace,
                                                   SchemaAutoUpdateCompatibilityStrategy strategy)
             throws PulsarAdminException;
+
+    /**
+     * Get schema validation enforced for namespace.
+     * @param namespace
+     * @param
+     * @return the schema validation enforced flag
+     * @throws NotAuthorizedException
```

Comment:
Can you please make sure the exceptions is aligned with the error responses returned at the web service?